src/avram/query_builder.cr

```diff
@@ -176,7 +183,7 @@ class Avram::QueryBuilder
   end
 
   def orders
-    @orders.uniq!(&.column)
+    @orders.uniq!(&.uid)
```

Comment:
Since `Avram::OrderByRandom` doesn't have a `column` method, I had to find a way to select unique order clauses. I'm not 100% sure if `uid` is the best method name here. Suggestions are welcome. :smile:

Reply:
Does this value matter for the random order? Is it just to ensure we don't have 2 `ORDER BY RANDOM()`? If that's the case, can we just make a `column` method on the random that returns `"*"`? I was thinking since you'll see "*" as meaning "all" that maybe that's the "column".... Not sure if that would work the same or not